druid-shell/src/common_util.rs

```diff
@@ -21,6 +21,7 @@ use std::sync::atomic::{AtomicU64, Ordering};
 /// Strip the access keys from the menu string.
 ///
 /// Changes "E&xit" to "Exit". Actual ampersands are escaped as "&&".
+#[allow(dead_code)]
 #[cfg(any(target_os = "macos", target_os = "linux"))]
 pub fn strip_access_key(raw_menu_text: &str) -> String {
```

Comment:
Seems like you can actually do
```rust
#[cfg(not(feature = "x11"))]
#[cfg(any(target_os = "macos", target_os = "linux"))]
pub fn strip_access_key(raw_menu_text: &str) -> String {
``` 
Which seems better to me than using `dead_code`

Reply:
Oh yeah, that's a lot nicer than the `#[cfg]` mess I made in a previous commit, haha. Done.